src/widget/scroll.rs

```diff
@@ -15,17 +15,23 @@
 //! A container that scrolls its contents.
 
 use std::f64::INFINITY;
+use std::time::{Duration, Instant};
 
 use log::error;
 
 use crate::{
     Action, BaseState, BoxConstraints, Data, Env, Event, EventCtx, LayoutCtx, PaintCtx, Point,
-    Rect, Size, UpdateCtx, Vec2, Widget, WidgetPod,
+    Rect, Size, TimerToken, UpdateCtx, Vec2, Widget, WidgetPod,
 };
 
-use crate::piet::RenderContext;
+use crate::piet::{Color, RenderContext};
 
-use crate::kurbo::Affine;
+use crate::kurbo::{Affine, RoundedRect};
+
+const SCROLL_BAR_WIDTH: f64 = 8.;
+const SCROLL_BAR_PAD: f64 = 2.;
+const SCROLL_BAR_BG: Color = Color::rgb8(255, 255, 255);
+const SCROLL_BAR_BORDER: Color = Color::rgba8(0, 0, 0, 76);
```

Comment:
It seems there is now a `theme.rs` file. Which values should live there? Only colors?

Reply:
The way I've been doing it is: colors in the theme, sizes in the theme only if they're standard sizes used by other widgets.

Reply:
Moved colors to the `theme.rs`